tarantool/crud.py

```diff
@@ -0,0 +1,81 @@
+"""
+This module provides API for interaction with a crud module.
```

Comment:
At first, it's definitely "the" crud module.

And we definitely need a link here.

Reply:
Done